### PR TITLE
Python functionality homework

### PR DESCRIPTION
**Pull request by sudakevych-andrii**

### CODE REVIEW

File: python_functionality/homework.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
+    for i in range(len(data)):
+        for redundant_key in redundant_keys:
+            del data[i][redundant_key]
+    return data
+
+
+def task_3_find_item_via_value(data: DT, value) -> DT:
+    for i in range(len(data)):
```
Comment (dimigor) [inline python_functionality/homework.py]: why U use len() to iterate over a list/dict?

File: python_functionality/homework.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        if 'name' in data[i]:
+            data[i]['name'] = data[i]['name'].capitalize()
+    return data
+
+
+def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
+    for i in range(len(data)):
+        for redundant_key in redundant_keys:
+            del data[i][redundant_key]
+    return data
```
Comment (dimigor) [inline python_functionality/homework.py]: try to solve these tasks using list/dict comp

File: python_functionality/homework.py (hunk 1 above)
Comment (sudakevych-andrii) [reply python_functionality/homework.py]: Done

File: python_functionality/homework.py (hunk 2 above)
Comment (sudakevych-andrii) [reply python_functionality/homework.py]: Done

**Pull request merged**